.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Unit tests - continuous integration
+
+on:
+  push:
+    branches:
+      - master
+      - b0.*
+  pull_request:
+    branches:
+      - master
+      - b0.*
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
don't we _also_ want to run it on different platforms? windows and macos?

Reply:
Fixed